Automatic Validation ended with:
> 2024-02-12T09:28:34.4410923Z ##[error] Manifest Error: The multi file manifest is incomplete.A multi file manifest must contain at least version, installer and defaultLocale manifest.

(Automated response - build 720.)

--- a/manifests/h/Hashicorp/Nomad/1.7.4/Hashicorp.Nomad.locale.en-US.yaml
+++ b/manifests/h/Hashicorp/Nomad/1.7.4/Hashicorp.Nomad.locale.en-US.yaml
@@ -22,5 +22,5 @@ Documentations:
   DocumentUrl: https://developer.hashicorp.com/nomad/docs
 - DocumentLabel: Tutorials
   DocumentUrl: https://developer.hashicorp.com/nomad/tutorials
-ManifestType: locale
+ManifestType: defaultLocale
 ManifestVersion: 1.6.0

